lib/sharepoint/client.rb

```diff
@@ -382,13 +402,11 @@ def update_metadata(filename, metadata, path, site_path = nil)
       prepared_metadata = prepare_metadata(metadata, __metadata['type'])
 
       easy = ethon_easy_json_requester
-      easy.headers = {
-        'accept' => 'application/json;odata=verbose',
-        'content-type' => 'application/json;odata=verbose',
-        'X-RequestDigest' => xrequest_digest(site_path),
-        'X-Http-Method' => 'PATCH',
-        'If-Match' => '*'
-      }
+      easy.headers = with_bearer_authentication_header({ 'accept' => 'application/json;odata=verbose',
+                                                         'content-type' => 'application/json;odata=verbose',
+                                                         'X-RequestDigest' => xrequest_digest(site_path),
+                                                         'X-Http-Method' => 'PATCH',
+                                                         'If-Match' => '*' })
```

Comment:
in order to reduce the number of lines changed, can we have the same indentation as above?